### PR TITLE
Decouple getUserFromJwt from Repository.js

### PR DESCRIPTION
**Pull request by bdougie**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [x] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description
#771 is blocked due to a test change. This fix will remove the dependency of having getUserFromJwt in the Repository.js. This will make the test more flexible.

## Related Tickets & Documents

#771 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/containers/Dashboard.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ function Repositories({user, match}) {
   return (
     <Suspense fallback={<Spinner />}>
       <Route exact path={match.url} render={(props) => <RepositoryGoals user={user} {...props} />} />
-      <Route path={"/repos/:repoOwner/:repoName/:id"} component={Repository} />
```
Comment (bdougie) [inline src/containers/Dashboard.js:11]: adds the user to the props of Repository

File: src/components/Repository.js [hunk 2]
```diff
@@ -98,8 +96,6 @@ function Repository({match}) {
       .finally(() => setIsForkLoading(false));
   };
 
-  const user = getUserFromJwt(Config.auth);
```
Comment (bdougie) [inline src/components/Repository.js:101]: user is now a props of Repository

Comment (bdougie) [reply src/components/Repository.js:101]: FYI, @filiptronicek. This should make you able to make your change without failing or updating the test.

Comment (filiptronicek) [reply src/components/Repository.js:101]: Awesome!

**Pull request merged**